lightning/src/chain/chainmonitor.rs

```diff
@@ -276,17 +276,16 @@ where C::Target: chain::Filter,
 				return Err(ChannelMonitorUpdateErr::PermanentFailure)},
 			hash_map::Entry::Vacant(e) => e,
 		};
+		let funding_txo = monitor.get_funding_txo();
```

Comment:
Perhaps just make this a `channel_id` variable so you don't convert twice.

Reply:
Actually, moved the whole thing inside the log macro call, that way it doesn't run at all if we're compiled without trace logging enabled.